spec.bs

```diff
@@ -2024,6 +2025,10 @@ To <dfn>build trusted bidding signals url</dfn> given a [=URL=] |signalsUrl|, an
 [=strings=] |keys|, an [=ordered set=] of [=strings=] |igNames|, an {{unsigned short}}-or-null
 |experimentGroupId|, and an [=origin=] |topLevelOrigin|:
 1. Let |queryParamsList| be a new empty [=list=].
+
+  Note: Create a query param of the form "`&<name>=<values in comma-delimited list>`", e.g.,
+    "`hostname=publisher1.com&keys=key1,key2&interestGroupNames=ad+platform,name2&experimentGroupId=1234`".
+
 1. [=list/Append=] "hostname=" to |queryParamsList|.
 1. [=list/Append=] the result of [=string/UTF-8 percent-encoding=] |topLevelOrigin| using

```

Comment:
Note that |topLevelOrigin| is an origin rather than a string, so you ought to serialize it before encoding it:

```suggestion
1. [=list/Append=] the result of [=string/UTF-8 percent-encoding=] the [=serialization of an
  origin|ASCII serialization of an origin|serialization=] of|topLevelOrigin| using
```

This isn't essential for this PR.